CryptoTechReminderSystem.Test/Gateway/SlackGatewayTests.cs

```diff
@@ -99,19 +145,58 @@ public void CanSendAPostMessageRequestWithAChannelAndText()
             {
                 var channel = "U98ZL999";
                 var text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
-                var message = new Message()
+                var message = new Message
                 {
                     Channel = channel,
                     Text = text
                 };
             
                 _slackGateway.Send(message);
             
-                var receivedRequest = _slackApi.ReceivedRequests.First();
+                var receivedRequest = _slackApi.ReceivedRequests().First();
                 
                 JObject.Parse(receivedRequest.RequestBody)["channel"].ToString().Should().Be(channel);
                 JObject.Parse(receivedRequest.RequestBody)["text"].ToString().Should().Be(text);
             }
+            
+            [Test]
+            public void CanSendAPostMessageRequestWithSuccess()
+            {
+                var channel = "U98ZL999";
```

Comment:
This variable and the `text` variable can each be extracted into `private const string`.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/37/commits/25f884a7f9ade5e77f0e29dc79bc4922c5c8cdf5